turbo.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "baseBranch": "origin/main",
+  "pipeline": {
+    "test": {
+      "outputs": []
+    },
+    "svelte-check": {
+      "outputs": []
+    }
```

Comment:
how does Turbo know when it should rerun those?

Reply:
It re-runs them when they have changed against origin/main

Reply:
But when **what** has changed?  🤔

Reply:
When the individual packages have changed, or their dependents. It hashes the contents of the source files committed to git within the individual package.

Reply:
clever, would it recognize a dependency on the root `tsconfig.json`~? can it be added manually?

Reply:
Yes, if anything changes in the root it'll blast everything